packages/nextcloud/test/webdav_test.dart

```diff
@@ -6,7 +6,7 @@ import 'package:nextcloud/nextcloud.dart';
 import 'package:nextcloud_test/nextcloud_test.dart';
 import 'package:test/test.dart';
 import 'package:test_api/src/backend/invoker.dart';
-import 'package:universal_io/io.dart';
+import 'package:universal_io/io.dart' hide HttpClient;
```

Comment:
Unnecessary hide

Reply:
I intentionally added it so we do not accidentally use it over the http package.
I wanted to show that the universal_io import should only be used for the `file`  features.

Should I change this?

Reply:
Yeah, I don't think it really matters.